src/wms/src/main/java/org/geoserver/wms/map/PDFMapResponse.java

```diff
@@ -47,6 +70,11 @@
     /** A logger for this class. */
     private static final Logger LOGGER = org.geotools.util.logging.Logging
             .getLogger("org.vfny.geoserver.responses.wms.map.pdf");
+    
+    /**
+     * Whether to apply the new vector hatch fill optimization, or not (on by default, this is just a safeguard)
+     */
```

Comment:
About the docs... unsure, the property is meant only as a safeguard, and there are a lot similarly undocumented.